src/Hng.Infrastructure/Migrations/20240730120752_Initial.cs

```diff
@@ -6,11 +6,23 @@
 namespace Hng.Infrastructure.Migrations
 {
     /// <inheritdoc />
-    public partial class newdb_setup : Migration
+    public partial class Initial : Migration
```

Comment:
Don't make changes to the old migration files, it would not tally with the one in the db in production and cause deployment errors, 
Add new migration file for your changes or added models instead